jaxrs-tck/pom.xml

```diff
@@ -78,6 +80,24 @@
         <tag>HEAD</tag>
     </scm>
 
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                    <source>11</source>
```

Comment:
You can omit these properties as you already gave that values in lines 33 and 34.